FireboltNETSDK/Client/FireboltCommand.cs

```diff
@@ -454,5 +456,72 @@ public override async Task<int> ExecuteNonQueryAsync(CancellationToken cancellat
             await ExecuteCommandAsync(StrictCommandText, cancellationToken);
             return await Task.FromResult(0);
         }
+
+        /// <summary>
+        /// Executes a query asynchronously on the server-side without returning results.
+        /// The token to track the query status can be accessed via the AsyncToken property.
+        /// </summary>
+        /// <returns>Always returns 0.</returns>
+        public int ExecuteAsyncNonQuery()
+        {
+            return ExecuteAsyncNonQueryAsync().GetAwaiter().GetResult();
+        }
+
+        /// <summary>
+        /// Executes a query asynchronously on the server-side without returning results.
+        /// The token to track the query status can be accessed via the AsyncToken property.
+        /// </summary>
+        /// <returns>A task representing the asynchronous operation. Always returns 0.</returns>
+        public async Task<int> ExecuteAsyncNonQueryAsync(CancellationToken cancellationToken = default)
```

Comment:
Investigate a way to reduce duplication with a sync counterpart by passing in parameter override. And a way to parse the response.